test/ci-op-configresolver-integration/run.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# This test runs the ci-operator-configresolver and verifies that it returns
+# properly resolved configs
+
+#===================================================================
+# FUNCTION trap_add ()
+#
+# Purpose:  prepends a command to a trap
+#
+# - 1st arg:  code to add
+# - remaining args:  names of traps to modify
+#
+# Example:  trap_add 'echo "in trap DEBUG"' DEBUG
+#
+# See: http://stackoverflow.com/questions/3338030/multiple-bash-traps-for-the-same-signal
+#===================================================================
+function trap_add() {
```

Comment:
Every time I see Bash like this I wince -- let's not use it. We can implement this more concisely